hm3.sample.ini

```diff
@@ -203,13 +203,24 @@ admin_users=
 ; By default Cypht uses the server name used in the request to determine
 ; the domain name to set the cookie for. Configurations that use a reverse
 ; proxy might need to define the domain name used for cookies. Leave this
-; blank to let Cypht auotmatically determine the domain. You can also use
+; blank to let Cypht automatically determine the domain. You can also use
 ; the special value of "none" to force Cypht to NOT set the cookie domain
 ; property at all. This is not recommended unless you know what you are
 ; doing!
 cookie_domain=
 
 
+; Cookie Path
+; -------------
+; By default Cypht uses the request URI to determine the cookie path to set
+; the cookie for. Configurations that use mod_rewrite might need to define
+; the path used for cookies. Leave this blank to let Cypht automatically
+; determine the path. You can also use the special value of "none" to force
+; Cypht to NOT set the cookie path property at all. This is not recommended
+; unless you know what you are doing!
```

Comment:
The example with `tiki-git` would be of help here as well :wink:.

Reply:
Sure, added one.